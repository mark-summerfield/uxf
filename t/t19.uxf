uxf 1.0
#<Suitable for SQLite conversion>
= PriceList Date Price Quantity ID Description
(PriceList
  2022-09-21 3.99 2 <CH1-A2> <Chisels (pair), 1in &amp; 1Â¼in> 
  2022-10-02 4.49 1 <HV2-K9> <Hammer, 2lb> 
  2022-10-02 1e1 1 <SX4-D1> <Eversure Sealant, 13-floz> 
)
