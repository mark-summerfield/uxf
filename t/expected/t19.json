{
  "UXF^table": {
    "name": "PriceList",
    "comment": null,
    "fields": {
      "Date": null,
      "Price": null,
      "Quantity": null,
      "ID": null,
      "Description": null
    },
    "records": [
      [
        {
          "UXF^date": "2022-09-21"
        },
        3.99,
        2,
        "CH1-A2",
        "Chisels (pair), 1in & 1\u00bcin"
      ],
      [
        {
          "UXF^date": "2022-10-02"
        },
        4.49,
        1,
        "HV2-K9",
        "Hammer, 2lb"
      ],
      [
        {
          "UXF^date": "2022-10-02"
        },
        5.89,
        1,
        "SX4-D1",
        "Eversure Sealant, 13-floz"
      ]
    ]
  }
}